front/src/utils/uiCoreHelpers.ts

```diff
@@ -0,0 +1,23 @@
+type SelectFixedOption =
+  | {
+      label: string;
+      id: string;
+    }
+  | undefined;
```

Comment:
Why allow `undefined` here?

Reply:
`sortedChOptions` can never contain an `undefined` item. Sorry, I don't understand the justification.

`undefined` items don't make sense to me, because `Select` has a separate mechanism to unset the selected option: placeholder.

Reply:
Actually that's not possible because of an osrd-ui bug: https://github.com/OpenRailAssociation/osrd-ui/pull/557